indico/modules/rb_new/client/js/components/TimeRangePicker.jsx

```diff
@@ -41,29 +41,65 @@ function _humanizeDuration(duration) {
     }
 }
 
+function generateStartTimeOptions(allowPastTimes) {
+    const options = [];
+    const end = moment().endOf('day');
+    const next = moment(START_HOUR, 'HH:mm');
+    let serializedNext;
```

Comment:
why not just make it const and declare it inside the if when assigning it?

Reply:
Parts of this function were copy-pasted. But I will take a look.